Extension/Plugin/PluginInterface.php

```diff
@@ -35,4 +35,14 @@
 interface PluginInterface
 {
     public function getType();
+
+    /**
+     * Get the plugin name.
+     */
+    public function getName();
```

Comment:
By adding these 2 new methods to the contract we force existing Plugins to implement these methods. Meaning it will break current plugins in SmartyBundle and those created by SmartyBundle users in their projects. It's just not visible in tests because there is no coverage atm for plugins :(